packages/ng/src/app/api/select/select-api.component.ts

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
Your namming implies a different architecture, it implies that `select-api` is a submodule of `select` while `api-select` is a submodule of `api`

also it would be more coherent with the `user-select`  which is a submodule of `user`